osi_common.proto

```diff
@@ -97,18 +97,24 @@ message Dimension3d
 // 
 // The preferred angular range is (-pi, pi]. The coordinate system is defined as right-handed. 
 //  
-// The rotations are to be performed \b roll \b first, \b pitch \b second and \b yaw \b third to follow the definition according to [1]. 
+// The rotations are to be performed \b yaw \b first, \b pitch \b second and \b roll \b third to follow the definition according to [1]. 
 // 
 // Rotations are defined in the reference coordinate frame around z-axis (=yaw), y-axis (=pitch) and x-axis (=roll), 
 // not in the body frame of the object. 
-// A positive angle corresponds to a counter-clockwise rotation around the respective axis.
+// A positive angle corresponds to a clockwise rotation around the respective axis.
```

Comment:
The terms "clockwise" or "counter-clockwise" are highly ambiguous and should thus be avoided! I propose the following unambiguous scientific definition, for which a lot of illustrative examples pop up when you make a Google search:
```For the sense of each rotation, the right-hand rule applies.```